guides/doc-Troubleshooting_Guide/master.adoc

```diff
@@ -0,0 +1,16 @@
+include::common/attributes.adoc[]
+:imagesdir: images
+include::common/header.adoc[]
+
+:deploying-on-aws:
+:context: troubleshooting-guide
+
+= Troubleshooting {ProjectName}
+
+include::common/modules/services.adoc[leveloffset=+1]
+
+include::common/modules/logging.adoc[leveloffset=+1]
+
+ifeval::["{build}" != "foreman-deb"]
```

Comment:
Why is this in your master.adoc?

Reply:
I was just copying stuff from the other guides